mod/deckorate_utility/set/all/wikirate/menu.rb

```diff
@@ -13,10 +13,11 @@ def menu_items
     [history_link, menu_edit_link, menu_board_link]
   end
 
-  def history_link text: ""
+  def history_link text: "", title: "History"
```

Comment:
I'm guessing the edit button code is in decko?  But this PR doesn't have the decko code that adds it?

Reply:
Correct!